examples/gzipped/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+
+	"github.com/erikdubbelboer/fasthttp"
+)
+
+func server() {
+	server := fasthttp.Server{
+		Name:              "Fasthttp server",
+		Handler:           handler,
+		ReduceMemoryUsage: true,
+	}
+	log.Fatal(server.ListenAndServe(":1313"))
+}
+
+func handler(ctx *fasthttp.RequestCtx) {
+	ctx.SetContentType("text/html")
+	ctx.Response.Header.Add("Content-Encoding", "gzip")
```

Comment:
We should not say that the Content-Encoding is gzip when we don't even know the client will accept gzip! (Move this one line below)